src/components/manage/productions/create/Event.js

```diff
@@ -59,6 +60,7 @@ const Event = ({
           <Text>
             {locationName} {locationCity}
           </Text>
+          {organizerName.length > 0 && <Text>{organizerName}</Text>}
```

Comment:
If I didn't do this `organizerName.length > 0` check, extra spacing would be visible because the `Text` would still be rendered (with empty content) and it would still render the space around it.

Reply:
Alternatively I could do this check inside `Text` itself, so it doesn't get rendered if the content is an empty string?

Reply:
`{organizerName && <Text>{organizerName}</Text>}` is enough here I think

Reply:
Okay I'll change that. We won't do the check in `Text` itself then? Because I think it's also an "issue" (minor) for other `Text`s in this component.

Reply:
It only works with either `organizerName.length > 0` or `!!organizerName`. Not sure which is preferable. It looks like it doesn't automatically cast to boolean, so we need to either do an explicit comparison or cast to boolean ourselves.

Reply:
`{!!organizerName && <Text>{organizerName}</Text>}`

This will fix the issue

Reply:
Yes that's what I said, but I don't know which is preferable? 😄

Reply:
!!organizerName is more commonly used in React I think